channels/packages/api-manager/develop/deployment-storageos-api-manager.yaml

```diff
@@ -47,7 +47,7 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 100Mi
+            memory: 190Mi
```

Comment:
out of interest - why 190 ?

Reply:
You have to do the same at api-manager repo. Thanks @Arau

Reply:
@Arau Do you want it to see only in the upcoming release, or we have to backport to 2.5/2.6?